frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="profile-main-info-wrapper d-flex flex-column">
+
+    <div class="profile-main-info-header info-container">
+        <div class="user-info-container d-flex flex-row">
+            <div class="user-avatar-container">
+                <img src="../../../../assets/images/default-profile-img.jpg" alt="User avatar" class="user-avatar">
+            </div>
+            <div class="user-info d-flex flex-column">
+                <p class="username">TatiHutii</p>
+                <p class="user-rank"><span>Rank</span>~5,000,000</p>
```

Comment:
Please make all items not hard coded

We should show all data from our interfaces

Reply:
Do you mean to create real interfaces? or just make it like 
`<p class="username">{{user.username}}</p>`
 `<p class="user-rank"><span>Rank ~</span>{{user.rank}}</p>`
and in component.ts fill that data with fake user interface?

Reply:
yes, real interface